app/assets/javascripts/controllers/footer.js

```diff
@@ -252,13 +274,15 @@ class FooterCtrl {
   }
 
   refreshData() {
-    this.isRefreshing = true;
+    this.setState({ isRefreshing: true });
     this.syncManager.sync({
       force: true,
       performIntegrityCheck: true
     }).then((response) => {
       this.$timeout(() => {
-        this.isRefreshing = false;
+        // delaying the status update makes the UI look smoother
+        // when syncing finishes fast.
+        this.setState({ isRefreshing: false });
       }, 200);
```

Comment:
Be sure to factor out constants like this into named constants 👍

Also inline comments should use the [JSDocs](https://jsdoc.app/about-getting-started.html) format: `/** */`. Or perhaps  `/* */`, as that appears to be treated differently.

Reply:
Which constants are you talking about? `200`? Also can you confirm this comment is correct? I didn't add the timeout myself, but since it took me a bit to understand why it was there I thought I'd document it.

Reply:
Yeah the 200 constant. And ha, yeah, I guess that's what it's for. Hard to remember now 😳